### ISSUE TITLE
1.20, 1.20.1 and 1.19.4 Sodium crashing?

### ISSUE THREAD

**Issue by Gavindt26** [jellysquid3/Sodium#1824]
### Version information

mc1.19.4

### Reproduction Steps

When i start it with either the normal launcher or feather client it'll start up but itll stay at the loading screen almost half way there.

### Crash Report file

info: CONNECTING TO SERVER undefined
info: [2023-06-18T09:09:00Z INFO  rust_launcher::start::bindings] server: None
info: [2023-06-18T09:09:00Z INFO  rust_launcher::start] Loading version
info: {"versionSize":221198016,"state":{"VersionLoading":true},"currentDownloadRate":0,"expectedTasks":6,"downloadSpeed":0,"downloaded":0}
info: [2023-06-18T09:09:00Z INFO  rust_launcher::start] Finished loading version
info: [2023-06-18T09:09:00Z INFO  rust_launcher::start] Finished moving mods
info: [2023-06-18T09:09:00Z INFO  rust_launcher::start] Preparing assets
[2023-06-18T09:09:00Z INFO  rust_launcher::start] Preparing JRE
info: {"versionSize":221198016,"state":{"Downloading":["Libraries","Assets","Jre","Mods"]},"currentDownloadRate":0,"expectedTasks":6,"downloadSpeed":0,"downloaded":79183}
info: [2023-06-18T09:09:00Z INFO  rust_launcher::start] Preparing libraries
info: [2023-06-18T09:09:00Z INFO  rust_launcher::start] Preparing mods
info: {"versionSize":221198016,"state":{"Downloading":["Libraries","Assets","Mods"]},"currentDownloadRate":0,"expectedTasks":6,"downloadSpeed":0,"downloaded":43227064}
info: {"versionSize":221198016,"state":{"Downloading":["Libraries","Assets","Mods"]},"currentDownloadRate":2137,"eta":"1 day","expectedTasks":6,"downloadSpeed":2137,"downloaded":518660682}
info: [2023-06-18T09:09:01Z INFO  rust_launcher::start] Finished preparing mods
info: {"versionSize":221198016,"state":{"Downloading":["Libraries","Assets"]},"currentDownloadRate":2137,"eta":"2 days","expectedTasks":6,"downloadSpeed":2137,"downloaded":563035006}
info: [2023-06-18T09:09:01Z INFO  rust_launcher::start] Finished preparing libraries
info: {"versionSize":221198016,"state":{"Downloading":["Assets"]},"currentDownloadRate":2137,"eta":"3 days","expectedTasks":6,"downloadSpeed":2137,"downloaded":691107653}
info: [2023-06-18T09:09:02Z INFO  rust_launcher::start] Finished preparing assets
[2023-06-18T09:09:02Z INFO  rust_launcher::start] Preparing registry
info: [2023-06-18T09:09:02Z INFO  rust_launcher::start] Finished preparing registry
info: {"versionSize":221198016,"state":{"Jvm":true},"currentDownloadRate":2137,"eta":"3 days","expectedTasks":6,"downloadSpeed":2137,"downloaded":808140216}
info: [2023-06-18T09:09:02Z INFO  rust_launcher::common::launch] starting JVM with arguments: ["-Djava.library.path=libraries/native\\net/digitalingot/fcef-windows/0.0.5\\extracted/;libraries/native\\net/digitalingot/fwebp-windows/0.0.1\\extracted/;libraries/native\\de/jcm/discord-game-sdk4j-windows/0.5.4\\extracted/;libraries/native\\org/jitsi/jnopus/1.0\\extracted/;libraries/native\\org/lwjgl/lwjgl/3.3.1\\extracted/;libraries/native\\org/lwjgl/lwjgl-jemalloc/3.3.1\\extracted/;libraries/native\\org/lwjgl/lwjgl-openal/3.3.1\\extracted/;libraries/native\\org/lwjgl/lwjgl-opengl/3.3.1\\extracted/;libraries/native\\org/lwjgl/lwjgl-glfw/3.3.1\\extracted/;libraries/native\\org/lwjgl/lwjgl-stb/3.3.1\\extracted/;libraries/native\\org/lwjgl/lwjgl-tinyfd/3.3.1\\extracted/;libraries/native\\com/mojang/text2speech/1.13.9\\extracted/", "--add-opens=java.desktop/java.awt.event=ALL-UNNAMED", "--add-opens=java.desktop/java.awt.color=ALL-UNNAMED", "--add-opens=java.desktop/java.awt=ALL-UNNAMED", "--add-opens=java.base/java.lang=ALL-UNNAMED", "-Xmx6340M", "-XX:+UnlockExperimentalVMOptions", "-XX:+UseG1GC", "-XX:G1NewSizePercent=20", "-XX:G1ReservePercent=20", "-XX:MaxGCPauseMillis=50", "-XX:G1HeapRegionSize=32M", "-Dlog4j2.formatMsgNoLookups=true", "-XX:ErrorFile=feather/java_error.log", "-Djavax.accessibility.assistive_technologies=", "-Djavax.net.ssl.trustStoreType=WINDOWS-ROOT", "-cp", "libraries\\java\\net/minecraft/client/1.19.4/minecraft-1.19.4.jar;libraries\\java\\net/fabricmc/tiny-mappings-parser/0.3.0+build.17/tiny-mappings-parser-0.3.0+build.17.jar;libraries\\java\\net/fabricmc/sponge-mixin/0.12.4+mixin.0.8.5/sponge-mixin-0.12.4+mixin.0.8.5.jar;libraries\\java\\net/fabricmc/tiny-remapper/0.8.2/tiny-remapper-0.8.2.jar;libraries\\java\\net/fabricmc/access-widener/2.1.0/access-widener-2.1.0.jar;libraries\\java\\org/ow2/asm/asm/9.4/asm-9.4.jar;libraries\\java\\org/ow2/asm/asm-analysis/9.4/asm-analysis-9.4.jar;libraries\\java\\org/ow2/asm/asm-commons/9.4/asm-commons-9.4.jar;libraries\\java\\org/ow2/asm/asm-tree/9.4/asm-tree-9.4.jar;libraries\\java\\org/ow2/asm/asm-util/9.4/asm-util-9.4.jar;libraries\\java\\net/fabricmc/intermediary/1.19.4/intermediary-1.19.4.jar;libraries/java\\net/fabricmc/fabric-loader/0.14.21/fabric-loader-0.14.21.jar;libraries\\java\\de/javagl/obj/0.3.0/obj-0.3.0.jar;libraries\\java\\de/jcm/discord-game-sdk4j/0.5.4/discord-game-sdk4j-0.5.4;libraries\\java\\net/digitalingot/feather-server-api-proto/1.1-SNAPSHOT/feather-server-api-proto-1.1-SNAPSHOT.jar;libraries\\java\\net/digitalingot/rust-extension/1.0.7/rust-extension-1.0.7.jar;libraries\\java\\net/digitalingot/fcef/0.0.5/fcef-0.0.5.jar;libraries\\java\\net/digitalingot/fwebp/0.0.1/fwebp-0.0.1.jar;libraries\\java\\net/digitalingot/feather-server-api/messaging/0.0.2/messaging-0.0.2-SNAPSHOT.jar;libraries\\java\\net/digitalingot/feather-server-api/api/0.0.2/api-0.0.2-SNAPSHOT.jar;libraries\\java\\org/jitsi/libjitsi/1.0-CUSTOM/libjitsi-1.0-CUSTOM.jar;libraries\\java\\org/jitsi/jitsi-utils/1.0-45-gc3afb76/jitsi-utils-1.0-45-gc3afb76.jar;libraries\\java\\org/capnproto/runtime/0.1.10/runtime-0.1.10.jar;libraries\\java\\com/google/inject/guice/5.1.0/guice-vendored-5.1.0.jar;libraries\\java\\javassist/javassist/3.12.1.GA/javassist-3.12.1.GA.jar;libraries\\java\\com/github/oshi/oshi-core/6.2.2/oshi-core-6.2.2.jar;libraries\\java\\com/google/code/gson/gson/2.10/gson-2.10.jar;libraries\\java\\com/google/guava/failureaccess/1.0.1/failureaccess-1.0.1.jar;libraries\\java\\com/google/guava/guava/31.1-jre/guava-31.1-jre.jar;libraries\\java\\com/ibm/icu/icu4j/71.1/icu4j-71.1.jar;libraries\\java\\com/mojang/authlib/3.18.38/authlib-3.18.38.jar;libraries\\java\\com/mojang/blocklist/1.0.10/blocklist-1.0.10.jar;libraries\\java\\com/mojang/brigadier/1.0.18/brigadier-1.0.18.jar;libraries\\java\\com/mojang/datafixerupper/6.0.6/datafixerupper-6.0.6.jar;libraries\\java\\com/mojang/javabridge/2.0.25/javabridge-2.0.25.jar;libraries\\java\\com/mojang/logging/1.1.1/logging-1.1.1.jar;libraries\\java\\com/mojang/patchy/2.2.10/patchy-2.2.10.jar;libraries\\java\\com/mojang/text2speech/1.13.9/text2speech-1.13.9.jar;libraries\\java\\commons-codec/commons-codec/1.15/commons-codec-1.15.jar;libraries\\java\\commons-io/commons-io/2.11.0/commons-io-2.11.0.jar;libraries\\java\\commons-logging/commons-logging/1.2/commons-logging-1.2.jar;libraries\\java\\io/netty/netty-codec-http/4.1.82.Final/netty-codec-http-4.1.82.Final.jar;libraries\\java\\io/netty/netty-buffer/4.1.82.Final/netty-buffer-4.1.82.Final.jar;libraries\\java\\io/netty/netty-codec/4.1.82.Final/netty-codec-4.1.82.Final.jar;libraries\\java\\io/netty/netty-common/4.1.82.Final/netty-common-4.1.82.Final.jar;libraries\\java\\io/netty/netty-handler/4.1.82.Final/netty-handler-4.1.82.Final.jar;libraries\\java\\io/netty/netty-resolver/4.1.82.Final/netty-resolver-4.1.82.Final.jar;libraries\\java\\io/netty/netty-transport-classes-epoll/4.1.82.Final/netty-transport-classes-epoll-4.1.82.Final.jar;libraries\\java\\io/netty/netty-transport-native-unix-common/4.1.82.Final/netty-transport-native-unix-common-4.1.82.Final.jar;libraries\\java\\io/netty/netty-transport/4.1.82.Final/netty-transport-4.1.82.Final.jar;libraries\\java\\it/unimi/dsi/fastutil/8.5.9/fastutil-8.5.9.jar;libraries\\java\\net/java/dev/jna/jna-platform/5.12.1/jna-platform-5.12.1.jar;libraries\\java\\net/java/dev/jna/jna/5.12.1/jna-5.12.1.jar;libraries\\java\\net/sf/jopt-simple/jopt-simple/5.0.4/jopt-simple-5.0.4.jar;libraries\\java\\org/apache/commons/commons-compress/1.21/commons-compress-1.21.jar;libraries\\java\\org/apache/commons/commons-lang3/3.12.0/commons-lang3-3.12.0.jar;libraries\\java\\org/apache/httpcomponents/httpclient/4.5.13/httpclient-4.5.13.jar;libraries\\java\\org/apache/httpcomponents/httpcore/4.4.15/httpcore-4.4.15.jar;libraries\\java\\org/apache/logging/log4j/log4j-api/2.19.0/log4j-api-2.19.0.jar;libraries\\java\\org/apache/logging/log4j/log4j-core/2.19.0/log4j-core-2.19.0.jar;libraries\\java\\org/apache/logging/log4j/log4j-slf4j2-impl/2.19.0/log4j-slf4j2-impl-2.19.0.jar;libraries\\java\\org/joml/joml/1.10.5/joml-1.10.5.jar;libraries\\java\\org/lwjgl/lwjgl-glfw/3.3.1/lwjgl-glfw-3.3.1.jar;libraries\\java\\org/lwjgl/lwjgl-jemalloc/3.3.1/lwjgl-jemalloc-3.3.1.jar;libraries\\java\\org/lwjgl/lwjgl-openal/3.3.1/lwjgl-openal-3.3.1.jar;libraries\\java\\org/lwjgl/lwjgl-opengl/3.3.1/lwjgl-opengl-3.3.1.jar;libraries\\java\\org/lwjgl/lwjgl-stb/3.3.1/lwjgl-stb-3.3.1.jar;libraries\\java\\org/lwjgl/lwjgl-tinyfd/3.3.1/lwjgl-tinyfd-3.3.1.jar;libraries\\java\\org/lwjgl/lwjgl/3.3.1/lwjgl-3.3.1.jar;libraries\\java\\org/slf4j/slf4j-api/2.0.1/slf4j-api-2.0.1.jar", "net.digitalingot.rustextension.ProxiedStart", "net.fabricmc.loader.launch.knot.KnotClient", "--username", "gavindt26", "--version", "1.19.4-feather", "--gameDir", "C:\\Users\\admin\\AppData\\Roaming/.minecraft", "--assetsDir", "assets", "--assetIndex", "1.19.4-feather-game", "--uuid", "b400ff1b16ef45b48cb67857be938bb3", "--accessToken", "<hidden>", "--userType", "msa", "--versionType", "feather"]
info: starting cleanup
info: sending javaw pid
info: [05:09:05] [main/INFO]: Loading Minecraft 1.19.4 with Fabric Loader 0.14.21
info: [05:09:05] [main/INFO]: Loading 63 mods:
	- controlling 11.0.0.2
	- entityculling 1.6.2-mc1.19.4
	- fabric-api 0.80.0+1.19.4
	   |-- fabric-api-base 0.4.25+504944c8f4
	   |-- fabric-api-lookup-api-v1 1.6.26+504944c8f4
	   |-- fabric-biome-api-v1 13.0.8+348a9c64f4
	   |-- fabric-block-api-v1 1.0.7+e022e5d1f4
	   |-- fabric-blockrenderlayer-v1 1.1.35+c2e6f674f4
	   |-- fabric-client-tags-api-v1 1.0.16+504944c8f4
	   |-- fabric-command-api-v1 1.2.28+f71b366ff4
	   |-- fabric-command-api-v2 2.2.7+504944c8f4
	   |-- fabric-commands-v0 0.2.45+df3654b3f4
	   |-- fabric-containers-v0 0.1.55+df3654b3f4
	   |-- fabric-content-registries-v0 3.5.10+ae0966baf4
	   |-- fabric-convention-tags-v1 1.4.2+9a7c5daaf4
	   |-- fabric-crash-report-info-v1 0.2.16+aeb40ebef4
	   |-- fabric-data-generation-api-v1 11.4.1+504944c8f4
	   |-- fabric-dimensions-v1 2.1.46+7f87f8faf4
	   |-- fabric-entity-events-v1 1.5.14+504944c8f4
	   |-- fabric-events-interaction-v0 0.5.0+76ba65ebf4
	   |-- fabric-events-lifecycle-v0 0.2.53+df3654b3f4
	   |-- fabric-game-rule-api-v1 1.0.34+a1ccd7bff4
	   |-- fabric-item-api-v1 2.1.18+504944c8f4
	   |-- fabric-item-group-api-v1 3.0.6+043f9acff4
	   |-- fabric-key-binding-api-v1 1.0.34+504944c8f4
	   |-- fabric-keybindings-v0 0.2.32+df3654b3f4
	   |-- fabric-lifecycle-events-v1 2.2.16+504944c8f4
	   |-- fabric-loot-api-v2 1.1.28+75e98211f4
	   |-- fabric-loot-tables-v1 1.1.32+9e7660c6f4
	   |-- fabric-message-api-v1 5.1.2+504944c8f4
	   |-- fabric-mining-level-api-v1 2.1.40+49abcf7ef4
	   |-- fabric-models-v0 0.3.31+504944c8f4
	   |-- fabric-networking-api-v1 1.3.2+504944c8f4
	   |-- fabric-networking-v0 0.3.42+df3654b3f4
	   |-- fabric-object-builder-api-v1 7.0.4+504944c8f4
	   |-- fabric-particles-v1 1.0.24+f1e4495bf4
	   |-- fabric-recipe-api-v1 1.0.9+a1ccd7bff4
	   |-- fabric-registry-sync-v0 2.2.1+504944c8f4
	   |-- fabric-renderer-api-v1 2.2.6+504944c8f4
	   |-- fabric-renderer-indigo 1.1.2+81e8c576f4
	   |-- fabric-renderer-registries-v1 3.2.39+df3654b3f4
	   |-- fabric-rendering-data-attachment-v1 0.3.29+afca2f3ef4
	   |-- fabric-rendering-fluids-v1 3.0.22+504944c8f4
	   |-- fabric-rendering-v0 1.1.42+df3654b3f4
	   |-- fabric-rendering-v1 2.1.2+504944c8f4
	   |-- fabric-resource-conditions-api-v1 2.3.2+e6c7d4eef4
	   |-- fabric-resource-loader-v0 0.11.3+1e1fb126f4
	   |-- fabric-screen-api-v1 1.0.46+504944c8f4
	   |-- fabric-screen-handler-api-v1 1.3.21+504944c8f4
	   |-- fabric-sound-api-v1 1.0.10+504944c8f4
	   |-- fabric-transfer-api-v1 3.1.2+504944c8f4
	   \-- fabric-transitive-access-wideners-v1 3.0.4+504944c8f4
	- fabricloader 0.14.21
	- feather 1.0.0-SNAPSHOT
	- iris 1.6.4
	   |-- io_github_douira_glsl-transformer 2.0.0-pre13
	   |-- org_anarres_jcpp 1.4.14
	   \-- org_antlr_antlr4-runtime 4.11.1
	- java 17
	- lithium 0.11.1
	- minecraft 1.19.4
	- sodium 0.4.10+build.24
	- starlight 1.1.1+fabric.ae22326
info: [05:09:05] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/C:/Users/admin/AppData/Roaming/.minecraft/libraries/java/net/fabricmc/sponge-mixin/0.12.4+mixin.0.8.5/sponge-mixin-0.12.4+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
info: [05:09:05] [main/INFO]: Compatibility level set to JAVA_17
info: [05:09:07] [main/INFO]: Loaded configuration file for Lithium: 114 options available, 0 override(s) found
[05:09:07] [main/INFO]: Loaded configuration file for Sodium: 33 options available, 3 override(s) found
[05:09:08] [main/WARN]: Error loading class: com/terraformersmc/modmenu/config/option/EnumConfigOption (java.lang.ClassNotFoundException: com/terraformersmc/modmenu/config/option/EnumConfigOption)
[05:09:08] [main/WARN]: Force-disabling mixin 'features.entity.fast_render.MixinCuboid' as rule 'mixin.features.entity.fast_render' (added by mods [iris]) disables it and children
[05:09:08] [main/WARN]: Force-disabling mixin 'features.entity.fast_render.MixinModelPart' as rule 'mixin.features.entity.fast_render' (added by mods [iris]) disables it and children
[05:09:08] [main/WARN]: Force-disabling mixin 'features.gui.font.MixinGlyphRenderer' as rule 'mixin.features.gui.font' (added by mods [iris]) disables it and children
[05:09:08] [main/WARN]: Force-disabling mixin 'features.sky.MixinWorldRenderer' as rule 'mixin.features.sky' (added by mods [iris]) disables it and children
[05:09:09] [main/WARN]: Error loading class: net/optifine/Config (java.lang.ClassNotFoundException: net/optifine/Config)
info: [05:09:25] [Render thread/INFO]: Launching a new process to probe the system configuration!
info: [05:09:25] [Render thread/INFO]: OpenGL Vendor: Intel
info: [05:09:25] [Render thread/INFO]: OpenGL Renderer: Intel(R) HD Graphics 2500
info: [05:09:25] [Render thread/INFO]: OpenGL Version: 3.3.0 - Build 10.18.10.4252
info: [05:09:26] [Render thread/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
info: [05:09:27] [Render thread/INFO]: Setting user: gavindt26
info: [05:09:27] [Render thread/INFO]: [Indigo] Different rendering plugin detected; not applying Indigo.
info: [05:09:28] [Render thread/INFO]: Backend library: LWJGL version 3.3.1 SNAPSHOT
info: [05:09:28] [ForkJoinPool.commonPool-worker-1/WARN]: [Iris Update Check] This version doesn't have an update index, skipping.
info: [05:09:29] [Render thread/INFO]: Debug functionality is disabled.
info: [05:09:29] [Render thread/INFO]: DSA support not detected.
info: [05:09:29] [Render thread/WARN]: Ignoring ambiguous boolean option GODRAYS
info: [05:09:29] [Render thread/WARN]: Ignoring ambiguous boolean option WAVING_LEAVES
info: [05:09:29] [Render thread/WARN]: Ignoring ambiguous boolean option WAVING_GRASS
info: [05:09:29] [Render thread/INFO]: Profile: Custom (+0 options changed by user)
info: [05:09:30] [Render thread/INFO]: Using shaderpack: Mr Butterness Lite Shader.zip
info: [05:09:30] [Render thread/WARN]: Static binding violation: PRIVATE @Overwrite method method_23182 in sodium.mixins.json:features.item.MixinItemRenderer from mod sodium cannot reduce visibiliy of PUBLIC target method, visibility will be upgraded.
info: [05:09:31] [Render thread/INFO]: Hardware information:
info: [05:09:31] [Render thread/INFO]: CPU: 4x Intel(R) Core(TM) i5-3470 CPU @ 3.20GHz
info: [05:09:31] [Render thread/INFO]: GPU: Intel(R) HD Graphics 2500 (Supports OpenGL 3.2.0 - Build 10.18.10.4252)
info: [05:09:31] [Render thread/INFO]: OS: Windows 10 (10.0)
info: [05:09:33] [Render thread/INFO]: Narrator library for x64 successfully loaded
info: [05:09:33] [Render thread/INFO]: Reloading ResourceManager: vanilla, Fabric Mods (Controlling For Fabric, EntityCulling-Fabric, Fabric API, Fabric API Base, Fabric API Lookup API (v1), Fabric Biome API (v1), Fabric Block API (v1), Fabric BlockRenderLayer Registration (v1), Fabric Client Tags, Fabric Command API (v1), Fabric Command API (v2), Fabric Commands (v0), Fabric Containers (v0), Fabric Content Registries (v0), Fabric Convention Tags, Fabric Crash Report Info (v1), Fabric Data Generation API (v1), Fabric Dimensions API (v1), Fabric Entity Events (v1), Fabric Events Interaction (v0), Fabric Events Lifecycle (v0), Fabric Game Rule API (v1), Fabric Item API (v1), Fabric Item Group API (v1), Fabric Key Binding API (v1), Fabric Key Bindings (v0), Fabric Lifecycle Events (v1), Fabric Loot API (v2), Fabric Loot Tables (v1), Fabric Message API (v1), Fabric Mining Level API (v1), Fabric Models (v0), Fabric Networking API (v1), Fabric Networking (v0), Fabric Object Builder API (v1), Fabric Particles (v1), Fabric Recipe API (v1), Fabric Registry Sync (v0), Fabric Renderer API (v1), Fabric Renderer - Indigo, Fabric Renderer Registries (v1), Fabric Rendering Data Attachment (v1), Fabric Rendering Fluids (v1), Fabric Rendering (v0), Fabric Rendering (v1), Fabric Resource Conditions API (v1), Fabric Resource Loader (v0), Fabric Screen API (v1), Fabric Screen Handler API (v1), Fabric Sound API (v1), Fabric Transfer API (v1), Fabric Transitive Access Wideners (v1), Fabric Loader, Feather Client, Iris, Lithium, Sodium, Starlight)
info: [05:09:34] [Render thread/INFO]: Creating pipeline for dimension OVERWORLD
info: [05:09:34] [Render thread/INFO]: Starting custom uniform resolving
info: [05:09:37] [Render thread/ERROR]: Failed to create shader rendering pipeline, disabling shaders!
java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 16
	at net.coderbot.iris.rendertarget.RenderTargets.get(RenderTargets.java:107) ~[iris-mc1.19.4-1.6.4.temp.jar:?]
	at net.coderbot.iris.postprocess.CompositeRenderer.<init>(CompositeRenderer.java:135) ~[iris-mc1.19.4-1.6.4.temp.jar:?]
	at net.coderbot.iris.pipeline.newshader.NewWorldRenderingPipeline.<init>(NewWorldRenderingPipeline.java:322) ~[iris-mc1.19.4-1.6.4.temp.jar:?]
	at net.coderbot.iris.Iris.createPipeline(Iris.java:676) ~[iris-mc1.19.4-1.6.4.temp.jar:?]
	at net.coderbot.iris.pipeline.PipelineManager.preparePipeline(PipelineManager.java:34) ~[iris-mc1.19.4-1.6.4.temp.jar:?]
	at net.coderbot.iris.Iris.onLoadingComplete(Iris.java:235) ~[iris-mc1.19.4-1.6.4.temp.jar:?]
	at net.minecraft.class_442.handler$bdl000$iris$showSodiumIncompatScreen(class_442.java:559) ~[client-intermediary.jar:?]
	at net.minecraft.class_442.method_25426(class_442.java:156) ~[client-intermediary.jar:?]
	at net.minecraft.class_437.method_25423(class_437.java:427) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1507(class_310.java:1079) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.<init>(class_310.java:681) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:198) ~[minecraft-1.19.4.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28) ~[fabric-loader-0.14.21.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at net.digitalingot.rustextension.ProxiedStart.main(ProxiedStart.java:16) ~[rust-extension-1.0.7.jar:?]
info: [FBD] Primary: Intel(R) HD Graphics. High: Intel(R) HD Graphics. Low: Intel(R) HD Graphics.
info: [05:09:41] [Render thread/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='GL error GL_INVALID_ENUM'
info: [FBD] [THAP] F10 32
info: [FBD] Hardware Acceleration: 0
info: [05:09:41] [Render thread/INFO]: [STDOUT]: initialize on Thread[Render thread,5,main] with library path libraries/native\net/digitalingot/fcef-windows/0.0.5\extracted/
info: [05:09:42] [nioEventLoopGroup-3-1/INFO]: Initialized rust channel communication. Communicating on port 50374.
info: [05:09:42] [Render thread/INFO]: Feather: Optifine dectected? false
info: [05:09:43] [Render thread/INFO]: Setting core backend token
info: [05:09:43] [Render thread/INFO]: Authenticated with core backend
info: [05:09:43] [nioEventLoopGroup-4-1/INFO]: Established Redstone connection to redstone.feathermc.com/103.163.186.206:1668
info: [05:09:43] [nioEventLoopGroup-4-1/INFO]: Authenticating with Redstone...
info: [05:09:43] [nioEventLoopGroup-4-1/INFO]: Authenticated with Redstone
info: [05:09:44] [nioEventLoopGroup-2-1/INFO]: Connected to chat server
info: [05:09:44] [Render thread/INFO]: Starting voice...
info: [05:09:44] [Render thread/INFO]: Connecting to voice server...
error: Jun 18, 2023 5:09:44 AM java.util.jar.Attributes read
WARNING: Duplicate name in Manifest: Bnd-LastModified.
Ensure that the manifest does not have duplicate entries, and
that blank lines separate individual sections in both your
manifest and in the META-INF/MANIFEST.MF entry in the jar file.
error: Jun 18, 2023 5:09:44 AM java.util.jar.Attributes read
WARNING: Duplicate name in Manifest: Build-Jdk.
Ensure that the manifest does not have duplicate entries, and
that blank lines separate individual sections in both your
manifest and in the META-INF/MANIFEST.MF entry in the jar file.
error: Jun 18, 2023 5:09:44 AM java.util.jar.Attributes read
WARNING: Duplicate name in Manifest: Built-By.
Ensure that the manifest does not have duplicate entries, and
that blank lines separate individual sections in both your
manifest and in the META-INF/MANIFEST.MF entry in the jar file.
error: Jun 18, 2023 5:09:44 AM java.util.jar.Attributes read
WARNING: Duplicate name in Manifest: Bundle-ManifestVersion.
Ensure that the manifest does not have duplicate entries, and
that blank lines separate individual sections in both your
manifest and in the META-INF/MANIFEST.MF entry in the jar file.
error: Jun 18, 2023 5:09:44 AM org.jitsi.utils.logging.LoggerImpl log
INFO: Loading library jnopus from java.library.path rather than bundled version
info: [05:09:44] [nioEventLoopGroup-5-1/INFO]: Authenticating with voice server
info: [05:09:44] [nioEventLoopGroup-5-1/INFO]: Authenticated with voice server
info: [05:09:44] [Render thread/INFO]: Started Feather ()
info: [05:09:45] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
info: [05:09:45] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
info: [05:09:45] [Render thread/WARN]: Missing sound for event: minecraft:music.overworld.cherry_grove
info: [05:09:45] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on Speakers / Headphones (Realtek High Definition Audio)
info: [05:09:45] [Render thread/INFO]: Sound engine started
info: [05:09:45] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
info: [05:09:45] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/signs.png-atlas
info: [05:09:45] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
info: [05:09:45] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
info: [05:09:45] [Render thread/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/armor_trims.png-atlas
info: [05:09:45] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
info: [05:09:45] [Render thread/INFO]: Created: 64x64x4 minecraft:textures/atlas/decorated_pot.png-atlas
info: [05:09:45] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
info: [05:09:45] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
info: [05:09:46] [Render thread/INFO]: [Cosmetics] Normal Feather Cape was created
info: [05:09:46] [Render thread/WARN]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
info: [05:09:46] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/particles.png-atlas
info: [05:09:46] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
info: [05:09:46] [Render thread/INFO]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
info: [2023-06-18T09:10:12Z ERROR rust_launcher::messaging::channel] error during messaging worker: An existing connection was forcibly closed by the remote host. (os error 10054)
info: [2023-06-18T09:10:13Z INFO  rust_launcher::start::bindings] launch stalled

### Additional information

i use either 1.19.4, 1.20, both don't work it'll just stop in the loading process.

**Comment by TheUserCreated** [comment]
Can you reproduce the issue without the feather client? Sodium does not support feather afaik

**Comment by Gavindt26** [comment]
no i cant it does the same as feather itll start up the minecraft and just start on the loading.

**Comment by Timolino456** [comment]
This sounds very much like a duplicate of #899. Have you tried updating your Intel drivers (via the Intel Website)?

**Comment by Gavindt26** [comment]
i think thats what it is but my pc is messed up so ill try to update it ive already tried ill try again.

**Comment by pajicadvance** [comment]
As mentioned above, duplicate of #899 . You have to update your Intel GPU drivers, otherwise the game won't launch.

**Comment by Gavindt26** [comment]
Hey it just started working it was my Intel drivers. thats been causing problems with alot of problems with my pc and it also works with feather client, i used Driver Easy to help me so that may be sum peoples problems but yeah its working it also fixed stuff with my pc so id like to say thank you so much for the help and ill prob never forget this honestly. :)

**Comment by ATHARV-YO** [comment]
Bro i'm also facing the same problem can u tell me how to update  intel  gpu drivers

**Comment by Phoenix-Starlight** [comment]
> Bro i'm also facing the same problem can u tell me how to update intel gpu drivers

Hey! Next time, ask at discord first for help, the community can help you there. Anyways, it depends a lot on some stuff... OEMs, etc. But you'd look for the GPU on Intel and download the driver. Sometimes the driver won't install due to OEM locking, so then you would have to uninstall the driver first (restart then), and then install. Or have to go through the `.inf` method in Device Manager. And then Windows might willy nilly rollback your driver.
Since we don't know anything in particular, if you need help, go to https://caffeinemc.net/discord.

**Comment by NightingaleZK** [comment]
I'm having a similar experience on my 1.20.1 Fabric modpack.
I have isolated my modpack by installing each mod one-by-one to ensure that Sodium was indeed the issue.
I implemented a Shader known as "Hysteria Shaders" by ElocinDev into my modpack, which requires Sodium and Iris Shaders in addition.
As soon as Sodium and Iris Shaders are placed into the modpack, the launcher takes a solid 3-4 minutes to boot up, freezes for roughly 30 seconds and crashes.
As soon as I removed Sodium (which automatically removes Iris Shaders) from my modpack on Curseforge, the modpack loads up perfectly and runs with 0 issue.
I have tried rolling back the versions of Sodium and Iris Shaders to the version released on 11/23/2023 to see if it was a version issue, but had the same outcome.
I looked online for a solution, in which everyone stated it was most like a intel GPU issue that required Drivers to be updated.
I always have my drivers updated as I'm on Windows 11, and went ahead and checked them both on their respective apps, Windows Update, and Device Manager; nothing required an update as everything was up-to-date currently.
This issue with Sodium is very recent for me as of a week ago, getting back into Minecraft with my brother who uses shaders and experiencing it not loading for me at all.
I used to use the shader "Insanity" when it came out just fine, so I don't know what has changed in the Sodium mod since then.
